static/page-templates/page-templates.json

```diff
@@ -163,6 +163,10 @@
 		{
 			"title": "rockfield",
 			"demo": "https://rockfielddemo.wordpress.com/"
+		},
+		{
+			"title": "seedlet-blocks",
+			"demo": "https://seedletblocksdemo.wordpress.com/"
```

Comment:
OK to add, but apparently it isn't used much anywhere: https://github.com/Automattic/wp-calypso/pull/42354#discussion_r426955475

Reply:
Yasss, of course. My bad :-)